hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -173,6 +173,12 @@ public final class OzoneConfigKeys {
           "ozone.s3.administrators";
   public static final String OZONE_S3_ADMINISTRATORS_GROUPS =
           "ozone.s3.administrators.groups";
+
+  public static final String OZONE_SUPER_READ_USERNAMES =
```

Comment:
I would suggest change the name to "ozone.readonly.administrators" and "ozone.readonly.administrators.groups"

Reply:
Can you change the variable name too? 
OZONE_SUPER_READ_USERNAMES -> OZONE_READONLY_ADMINISTRATORS
OZONE_SUPER_READ_GROUPS -> OZONE_READONLY_ADMINISTRATORS_GROUPS